deeprankcore/domain/forcefield/__init__.py

```diff
@@ -58,17 +59,12 @@ def _find_matching_residue_class(self, residue: Residue):
         return None
 
     def get_vanderwaals_parameters(self, atom: Atom):
-        type_ = self._get_type(atom)
-
-        return self._vanderwaals_parameters[type_]
-
-    def _get_type(self, atom: Atom):
         atom_name = atom.name
 
         if atom.residue.amino_acid is None:
-            raise UnknownAtomError(f"no amino acid for {atom}")
-
-        residue_name = atom.residue.amino_acid.three_letter_code
```

Comment:
this and below UnknownAtomErrors used to be caught in the code that calls the functions. I just substituted the warnings raised there in here.